primitives/account/src/account/staking_contract/staker.rs

```diff
@@ -21,11 +22,13 @@ use crate::{
 };
 
 /// Struct representing a staker in the staking contract.
-/// The staker's balance is divided into active and inactive stake.
+/// The staker's balance is divided into active, inactive and retired stake.
 ///
 /// Actions concerning a staker are:
 /// 1. Create:           Creates a staker.
-/// 2. Stake:            Adds coins from any outside address to a staker's active balance.
+/// 2. AddStake:         Adds coins from any outside address to a staker's active balance.
+///                      This action is only possible if:
+///                        (a) if the resulting non retired funds respect the invariant 1 - minimum stake for non retired funds.
```

Comment:
Double "if" (already present in the intro above).
```suggestion
///                        (a) the resulting non-retired funds respect the invariant 1 - minimum stake for non-retired funds.
```